### DISCUSSION TITLE
I cant access to my external and internal storage

### DISCUSSION THREAD

**Discussion by AdrielXXO** [nathaneltitane/dextop#31] [General]
Hello!
First... thank you so much for this proyect! Its awesome!

But i have a problem. I cant access to my internal and external storage. I put a ".storage" file in the internal and external. but doesnt work.

**Comment by nathaneltitane** [comment]
Thank you for using Dextop!

If I recall, the .storage file is optional, but I will look into it.

What kind of device are you installing Dextop onto?

Could you please share your /storage location details using termux?

Thanks

**Reply by AdrielXXO** [reply]
Thanks to you for this!

I have already solved! She was not placing the file name correctly. That's the problem. I had skipped the letter "s" in the ".storage" file.

I have another problem... maybe you can help me. I need to install a webbrowser.
